umcontroller/umcontroller.go

```diff
@@ -857,6 +845,17 @@ func (umCtrl *Controller) createStateMachine() (string, []fsm.EventDesc, map[str
 			{Name: evUpdateStatusUpdated, Src: []string{stateStartRevert}, Dst: stateUpdateUmStatusOnRevert},
 			{Name: evContinue, Src: []string{stateUpdateUmStatusOnRevert}, Dst: stateStartRevert},
 			{Name: evSystemReverted, Src: []string{stateStartRevert}, Dst: stateIdle},
+			// process close connection
+			{Name: evConnectionClose, Src: []string{stateIdle}, Dst: stateInit},
```

Comment:
why not  
`{Name: evConnectionClose, Src: []string{stateIdle, statePrepareUpdate, ....}, Dst: stateInit},`

Reply:
I missed that. changed